tc-tests-utils.sh

```diff
@@ -299,14 +341,53 @@ generic_download_tarxz()
 
   mkdir -p ${target_dir} || true
 
-  wget ${url} -O - | pixz -d | tar -C ${target_dir} -xf -
+  wget ${url} -O - | ${UNXZ} | ${TAR} -C ${target_dir} -xf -
 }
 
 download_native_client_files()
 {
   generic_download_tarxz "$1" "${DEEPSPEECH_ARTIFACTS_ROOT}/native_client.tar.xz"
 }
 
+install_nuget()
+{
+  PROJECT_NAME=$1
+  if [ -z "${PROJECT_NAME}" ]; then
+    exit "Please call with a valid PROJECT_NAME"
+    exit 1
+  fi;
+
+  nuget="${PROJECT_NAME}.${DS_VERSION}.nupkg"
+
+  export PATH=$PATH:$(cygpath ${ChocolateyInstall})/bin
+
+  mkdir -p "${TASKCLUSTER_TMP_DIR}/repo/"
+  mkdir -p "${TASKCLUSTER_TMP_DIR}/ds/"
+
+  wget -O - "${DEEPSPEECH_ARTIFACTS_ROOT}/${nuget}" | gunzip > "${TASKCLUSTER_TMP_DIR}/${PROJECT_NAME}.${DS_VERSION}.nupkg"
```

Comment:
`gunzip` because TaskCluster infra does `gzip` and it looks like `wget` does not respect / deflate automagically